recipes/sdsl-lite/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sdsl-lite" %}
+{% set version = "2.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/simongog/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz.offline.install.gz
+  fn: {{ name }}.tar.gz # necessary because of the nonstandard file extension from upstream
+  sha256: 2f192977b47406ae8992292c7e841ef23d4656bf72f6140540bed53af68e06ed
+
+build:
+  number: 0
+  script: "$SRC_DIR/install.sh $PREFIX"  # [unix]
+  script: "$SRC_DIR/install.bat $PREFIX"  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libsdsl.a  # [unix]
+    - if not exist %LIBRARY_LIB%\\libsdsl.lib exit 1  # [win]
+
+about:
+  home: https://github.com/simongog/sdsl-lite
+  license: GPL-3
+  license_family: GPL
```

Comment:
Please, add the ``license_file``
https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
done